package.json

```diff
@@ -53,7 +53,7 @@
   },
   "scripts": {
     "analyze": "NODE_ENV=production NETWORK=main ANALYZE=1 node --max_old_space_size=4096 node_modules/.bin/webpack --config webpack.prod.config.js --progress --colors",
-    "build": "webpack --config webpack.prod.config.js",
+    "build": "node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --config webpack.prod.config.js",
```

Comment:
this does not seems to solve the build issue ?
why this change ?

Reply:
That is an attempt to fix the review app build problem and also the ability to build for prod locally.  It fixes the ability to build for prod locally, but doesn't seem to help with the review app build.